packages/web-components/fast-foundation/src/progress/progress.template.ts

```diff
@@ -15,7 +15,7 @@ export const ProgressTemplate: ViewTemplate<BaseProgress> = html`
         class="${x => (x.paused ? "paused" : "")}"
     >
         ${when(
-            x => x.value,
+            x => typeof x.value === "number" && x.value !== null,
```

Comment:
`null` will never be a `number` so you can just do:

```suggestion
            x => typeof x.value === "number",
```